auth/services/oauth/oauth.go

```diff
@@ -273,6 +274,12 @@ func (s *service) CreateJwtGrant(request services.CreateJwtGrantRequest) (*servi
 		return nil, err
 	}
 
+	for _, verifiableCredential := range request.Credentials {
```

Comment:
I think these validation steps are missing (chapter 5.2.1.7 of RFC003):

> The credential issuer equals the sub field of the JWT.
> The credential credentialSubject.id equals the iss field of the JWT.

Reply:
Note that this PR is based on: https://github.com/nuts-foundation/nuts-specification/pull/119 where any verifiable credential can be used. However, we could validate known types (such as `NutsAuthorizationCredential`)?

Reply:
Well it must be validated _somewhere_, maybe we can have specialised validators per credential type?

Reply:
I think we already have something like that. I'll take a look!